src/views/AuthorityView/AuthorityView.js

```diff
@@ -74,6 +72,31 @@ const AuthorityView = ({
     [location.search],
   );
 
+  const { deleteItem } = useAuthorityDelete({
+    onSettled: () => setRequestDeletion(false),
+    onError: () => {
+      const message = (
+        <FormattedMessage
+          id="ui-marc-authorities.authority-record.delete.error"
+          values={{ id: <strong>{authority.data.headingRef}</strong> }}
```

Comment:
Values should be an object containing string values that are referenced in a translation
`<strong>` should be placed inside translations in en.json instead
Also, I think it's better to rename `id` to `headingRef` because that's what we're passing to the translation